pages/checkout/index.js

```diff
@@ -88,24 +91,83 @@ export default function Checkout() {
         line_items: lineItems,
         shipping_lines: [],
       };
+    } else {
+      const data = {
+        payment_method: 'paypal',
+        payment_method_title: 'PayPal Standard',
+        set_paid: true,
+        billing: customerData,
+        shipping: customerData,
+        line_items: lineItems,
+        shipping_lines: [],
+      };
+    }
 
-      try {
-        const res = await fetch('./api/createOrder', {
-          method: 'POST',
-          headers: {
-            'Content-Type': 'application/json',
-          },
-          body: JSON.stringify(data),
-        });
-
-        if (res.status === 200 || res.status === 201) {
-          router.push('/checkout/success');
-        } else {
-          router.push('/checkout/failure');
-        }
-      } catch (err) {
-        console.log('Error Frontend: ', err);
+    try {
+      const res = await fetch('./api/createOrder', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(data),
+      });
+
+      if (res.status === 200 || res.status === 201) {
+        router.push('/checkout/success');
+      } else {
+        router.push('/checkout/failure');
       }
+    } catch (err) {
+      console.log('Error Frontend: ', err);
+    }
+  };
+
+  const onProcess = () => {
+    setProcessingOrder(true);
+  };
+
+  const onSuccess = async () => {
+    const customerData = {
+      first_name: firstName,
+      last_name: surname,
+      address_1: address,
+      address_2: '',
+      city: city,
+      state: province,
+      postcode: '0000',
+      country: country,
+      email: email,
+      phone: phoneNumber,
+    };
+
+    const lineItems = cartItems.map((item) => ({ product_id: item.product_id, variation_id: item.id, quantity: item.quantity }));
+
+    const data = {
+      payment_method: 'paypal',
+      payment_method_title: 'PayPal Standard',
+      set_paid: true,
+      billing: customerData,
+      shipping: customerData,
+      line_items: lineItems,
+      shipping_lines: [],
+    };
+
+    try {
+      const res = await fetch('./api/createOrder', {
```

Comment:
Consider writing a `fetch` helper util somewhere, so that you can just pass the endpoint, method, and body instead of having to correctly format it everywhere.